src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v2/validator/YFlowSharedEndpointEncapsulationValidator.java

```diff
@@ -51,8 +51,8 @@ private static Optional<String> validateBaseFlowEndpointV2VlanId(
         }
 
         //validate flowSharedEndpointEncapsulation vlanId is not more than 4095
-        if (flowSharedEndpointEncapsulation.getVlanId() > 4094) {
-            return Optional.of("VlanId must be less than 4095");
+        if (flowSharedEndpointEncapsulation.getVlanId() > 4095) {
```

Comment:
It might be better if we create a constant for this max port value
something like this one:

`STATS_VLAN_UPPER_BOUND = 4095`